samples/highcharts/demo/bar-chart/demo.js

```diff
@@ -41,20 +40,20 @@ Highcharts.chart('container', {
             groupPadding: 0.1
         }
     },
+
     legend: {
         layout: 'vertical',
         align: 'right',
         verticalAlign: 'top',
-        x: -40,
+        x: -20,
         y: 80,
-        floating: true,
-        borderWidth: 1,
-        backgroundColor: 'var(--highcharts-background-color, #ffffff)',
-        shadow: true
+        floating: true
```

Comment:
We need to keep the background, otherwise the grid lines cut right through the legend. Can be seen in the visual diff.
```suggestion
        floating: true,
        backgroundColor: 'var(--highcharts-background-color, #ffffff)'
```